src/app/tab-preferences/tab-preferences.page.ts

```diff
@@ -121,7 +125,21 @@ export class Tab3Page {
       this.allTestNetworks = result
     })
 
-    App.getInfo().then((result) => this.appVersion = result.version)
+    App.getInfo().then((result) => {
+      this.appVersion = result.version
+    })
+
+    this.flavor.isBetaFlavor().then((result) => {
+      if (result) {
+        this.package = "INTERNAL BETA"
+      }
+    })
+
+    this.flavor.isNoGoogleFlavor().then((result) => {
+      if (result) {
+        this.package = "Googleless Edition"
+      }
+    })
```

Comment:
Since noGoogle overrides beta, it would be faster to do something like
`if (isNoGoogle) {
    // noGoogle handling
} else if (isBeta) {
    // beta handling
}`

Reply:
I have no strong opinion on this. I kinda like it for its simplicity and copy-paste-ability, since both are async and exclusive to each other I see no performance penalties or other weird interactions.